src/Detector/DetectorLibrary/DetectorLibrary.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
How are you testing this library? I am not seeing it being referenced within `BuildScriptGenerator.csproj`?